docs/General/Meetings/project_tracking.md

```diff
@@ -0,0 +1,68 @@
+# Project Tracking
+
+## For the first 10-15 minutes of every lab meeting, the lab discusses the progress of the projects.
+
+When it is your turn to present, you **must** include a copy of [this slide](https://docs.google.com/presentation/d/1Gk5uMTpn8XCMvXmoP8AWjW3x-QM0mHfPB1BfteBfqK8/edit#slide=id.g2684830fade_0_0) at the beginning of your slide deck. 
+We will be using it to go through the progress of the projects.
+
+We use four different spreadsheets to track updates. Each of them will be linked in their subsection below. 
+
+### To Dos & Deadlines for BHK
+If there are any upcoming tasks or deadlines that you need Ben to address, [this spreadsheet](https://docs.google.com/spreadsheets/d/1gnfwpzSho3ePF1WhFLZC8aGuDp9IG-mM6GMHbURjy8A/edit?gid=0#gid=0) is where you add your requests.
+The following are the columns in the spreadsheet:
+
+- **Task**: Add a general descriptive title for the task.
+- **Deadline**: Add the deadline for this task.
+- **BHK review deadline**: Add the deadline for when you need Ben to review the task.
+- **Lab Members**: The lab members involved in this task. (If it's just yourself, only write your name. If it's multiple people, write all their names.)
+- **Comments**: Add any additional comments that you think are necessary.
+- **Relevant links**: Add any relevant links that are necessary for this task.
+
+
+### Manuscript Progress
+If you have any updates on the progress of your manuscript, [this spreadsheet](https://docs.google.com/spreadsheets/d/1dZcNEmIGCdFthwj5cskuATfDTJ-7Mup8D5gO_HrmIPE/edit?usp=sharing) is where you add your updates.
+The following are the columns in the spreadsheet:
+- **Lead Author (Mandatory)**: Add the name of the lead author of the manuscript. There can be multiple names if there are multiple lead authors working on this manuscript.
```

Comment:
add a line before the bullet points to show them as bullet points on the website
```suggestion
The following are the columns in the spreadsheet:

- **Lead Author (Mandatory)**: Add the name of the lead author of the manuscript. There can be multiple names if there are multiple lead authors working on this manuscript.
```